.gitattributes

```diff
@@ -0,0 +1,9 @@
+/tests export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+.php_cs.dist export-ignore
+.travis.yml export-ignore
+phpstan-baseline.neon
```

Comment:
Don't we need the `export-ignore` part here and on the following lines too?

Reply:
you're right :man_facepalming: